bitnami/contour/templates/crd-ingressroutes.yaml

```diff
@@ -1,381 +0,0 @@
-{{- if and .Values.contour.enabled .Values.contour.createCustomResource }}
```

Comment:
I think I should rename the keyword `createCustomResource` to `installCRDs` like the option of the [kong chart](https://github.com/bitnami/charts/blob/master/bitnami/kong/values.yaml#L142) since these are definitions and not an instance of a definition.

Reply:
Agree!